src/composer.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'll move it back, it was due to my lack of knowledge of correct php file structure.